### DISCUSSION TITLE
Animated WEBP to GIF

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7623] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

I have an animated webp file that is 2.3MB. How do I convert to a gif one while maintaining or lower the file size? I don't care about the quality.

**Comment by snibgo** [comment]
Please post a link to an animated webp file.

**Reply by Jay123210599** [reply]
https://drive.google.com/file/d/1qnkj8BQVV6UyN4obDd0jSgG8Ew674tip/view?usp=sharing

**Comment by snibgo** [comment]
```
magick nm.webp out.gif
```
We can get some compression with:
```
magick nm.webp -layers optimize out.gif
```

**Reply by Jay123210599** [reply]
Can I adjust the compression level?

**Comment by snibgo** [comment]
Various options effect the output file size. For example:
```
magick mn.webp -strip -resize 50% +remap -fuzz 25% -layers optimize-transparency out.gif
```

**Reply by Jay123210599** [reply]
what does strip, remap, and fuzz do?

**Comment by snibgo** [comment]
See https://imagemagick.org/script/command-line-options.php#strip and other options on that page.
